manifests/b/BandicamCompany/Bandicam2024/7.1.6.2461/BandicamCompany.Bandicam2024.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: BandicamCompany.Bandicam2024

```

```suggestion
PackageIdentifier: BandicamCompany.Bandicam
```
Please update file name and path to match this change.